### ISSUE TITLE
Extension does not work

### ISSUE THREAD

**Issue by femto-code** [da-the-dev/ExEx#3]
Every operation where the extension is supposed to modify the global/workspace `state.vscdb` file an error occurs:

![image](https://user-images.githubusercontent.com/59444046/176131396-fe6734cd-03c6-4dca-b1f6-493ac506d7ed.png)

While this happens on both VSCode and VSCodium it seems that VSCodium seems to be excluded if you take a look at how the path to the user data directory is generated. There are designated API calls to get these paths.

**Comment by femto-code** [comment]
I will probably create a fork and create a few PRs if I have enough time. I really like the idea of this extension!

**Comment by da-the-dev** [comment]
Yeah, I haven't worked on this extension for a long while because of uni. I hope I can get my hands on fixing it soon. Any help is welcome, though!

**Comment by femto-code** [comment]
FYI, I think VSCode [supports profiles natively now](https://code.visualstudio.com/docs/editor/profiles).

**Comment by da-the-dev** [comment]
> FYI, I think VSCode [supports profiles natively now](https://code.visualstudio.com/docs/editor/profiles).

Damn, this is basically ExEx, but native. Thank you for letting me know! I think I might archive this repo now.
